qcodes/instrument_drivers/agilent/Agilent_34400A.py

```diff
@@ -4,11 +4,11 @@
 
 class Agilent_34400A(VisaInstrument):
     """
-    This is the qcodes driver for the Agilent_34400A DMM Series,
-    tested with Agilent_34401A, Agilent_34410A, and Agilent_34411A
+    This is the QCoDeS driver for the Agilent_34400A DMM Series,
+    tested with Agilent_34401A, Agilent_34410A, and Agilent_34411A.
     """
 
-    def __init__(self, name, address, **kwargs):
+    def __init__(self, name: str, address: str, **kwargs) -> None:
```

Comment:
For future references: You are allowed to omit the None return type for init methods

Reply:
Sure, I am just adding them for general consistency in the framework.